docs/userguide.md

```diff
@@ -232,6 +233,38 @@ NAME   SHORTNAMES   APIGROUP   NAMESPACED   KIND
 Verifying the API type exists on all member clusters will ensure successful
 propagation to that cluster.
 
+### Enabling an API type in a new federation group
+When `kubefedctl enable` is used to enable types whose plural names (e.g. **deployments**.extensions
+and **deployments**.apps) match, the crd name of the generated federated type would also match (e.g.
+**deployments**.types.federation.k8s.io).
+
+`kubefedctl enable --federation-group string` specifies the name of the API group to use for the
+generated federation type. It is `types.federation.k8s.io` by default. If a new federation group is
+enabled, RBAC for federation controller should be updated accordingly.
```

Comment:
`s/ RBAC for federation controller should be updated accordingly/the RBAC permissions for the federation controller manager will need to be updated to include permissions for the new group/`